modules/migration-adding-replication-repository-to-cam.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// migration/migrating_openshift_3_to_4/migrating-openshift-3-to-4.adoc
+[id='migration-adding-replication-repository-to-cam_{context}']
+= Adding a replication repository to the CAM web console
+
+You can add an replication repository to the CAM web console.
+
+.Prerequisites
+
+* The replication repository must support the S3 API.
++
+[NOTE]
+====
+You can deploy a local S3 object storage with the link:https://github.com/fusor/mig-operator/blob/release-1.0/docs/usage/ObjectStorage.md[upstream NooBaa project or AWS S3].
```

Comment:
@jwmatthews 

Was the final decision to keep this link to github and the upstream NooBaa project?

Reply:
Yes, final decision is to continue with this github link.

This is a temporary path, for next ~month or so we will continue to point to this upstream documentation.
Once OCS is officially released we plan to switch to using downstream NooBaa